tests/test_basic.py

```diff
@@ -8,6 +8,10 @@
 import pandas as pd
 import tarfile
 
+# no need to log on W&B
+os.environ["WANDB_MODE"] = "offline"
```

Comment:
can you add to the readme what needs to be done to enable wandb logging to wandb.ai ?

Reply:
It will always be active and would prompt the user to be anonymous or linked to an account (optional) the first time.

Reply:
Ah yes right, this is only for the tests, make sense

Reply:
It can always be deactivated with environment variable like in the test